tools/jest-global-mocks.ts

```diff
@@ -1,42 +1,42 @@
+// Polyfill `window.crypto`
+import 'window-crypto';
+
 // tslint:disable no-any no-unsafe-any
 const mock = () => {
   let storage: Record<string, any> = {};
   return {
-    getItem: (key: string): any => key in storage ? storage[key] : null,
+    getItem: (key: string): any => (key in storage ? storage[key] : null),
     setItem: (key: string, value: any) => {
       storage[key] = value || '';
       return storage[key];
     },
     removeItem: (key: string) => delete storage[key],
-    clear: () => storage = {},
+    clear: () => (storage = {}),
```

Comment:
Are the parenthesis needed or for better readability?

Reply:
Kind of. Creating an assignment in a return statement can be confusing so we have a lint rule warning about this.

The `()` is just us being explicit that something is happening inside, then it returns.

Reply:
A better way for me to phrase it: It is not more readable, but the intent should be more clear.